src/rich_click/__main__.py

```diff
@@ -0,0 +1,15 @@
+"""
+Entry-point module, in case you use `python -m griffe`.
+
+Why does this file exist, and why `__main__`? For more info, read:
+
+- https://www.python.org/dev/peps/pep-0338/
+- https://docs.python.org/3/using/cmdline.html#cmdoption-m
+"""
+
+import sys
+
+from rich_click.cli import main
+
+if __name__ == "__main__":
+    sys.exit(main(sys.argv[1:]))
```

Comment:
The `sys.exit` is probably not useful as I believe Click commands exit themselves?

Reply:
They do unless you set `standalone_mode` to `False`, then exceptions are raised (I've learnt a lot about click in the process of writing this project!)

Reply:
Then I guess it's safer to keep the sys.exit here :slightly_smiling_face:

Reply:
Oh wait, this is the main function of rich-click itself, so my first comment is irrelevant. I'll leave sys.exit since we use argparse.

Reply:
Maybe - I sent this reply from my phone without seeing the code in context just before bed though, so it's not fully thought through 😅 But yeah, if it works without the `sys.exit` then I think it's probably fine to remove it 👍🏻

Reply:
Wait again, I'm getting confused :joy: My first comment *is* relevant because the main function runs a Click function which in turn either exits or raises. So yeah, removing sys.exit :rofl:

Reply:
![confused](https://media.giphy.com/media/3o7btPCcdNniyf0ArS/giphy-downsized.gif)

I'm a bit lost now 😆